### PR TITLE
Merge `dev-pt` into `main`

### PR DESCRIPTION
**Pull request by edsoncelio**
Changes
* Merge `dev-pt` into `main` to add a few definitions to live

### CODE REVIEW

**Comment by edsoncelio** [comment]
Hi @jihoon-seo or @seokho-son this a PR to go live (`dev-pt -> main`) with some new definitions localization in `pt-br`. I don’t know if this need to be reviewed by one of the maintainers, I'll add the label `maintainers` because I'm in doubt about that, thanks :)

**Review by brunoguidone — APPROVED** [review]
LGTM

**Review by jihoon-seo — COMMENTED** [review]
@edsoncelio Thank you for your work. 😊

Here is a one thing:
in `en` directory, `versioncontrol.md` is renamed to [`version-control.md`](https://github.com/cncf/glossary/blob/main/content/en/version-control.md).
So, please do so in `dev-pt : content/pt-br` as well. 😊

**Comment by edsoncelio** [comment]
> @edsoncelio Thank you for your work. 😊
> 
> Here is a one thing: in `en` directory, `versioncontrol.md` is renamed to [`version-control.md`](https://github.com/cncf/glossary/blob/main/content/en/version-control.md). So, please do so in `dev-pt : content/pt-br` as well. 😊

Thank you @jihoon-seo!
I've opened a PR for that: https://github.com/cncf/glossary/pull/1019

**Comment by edsoncelio** [comment]
PR https://github.com/cncf/glossary/pull/1019 merged 🎉

**Review by seokho-son — APPROVED** [review]
LGTM :)

File: content/pt-br/cloud-native-tech.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Tecnologia Nativa da Nuvem
+status: Completed
+category: conceito
+---
+
+## O que é
+
+As tecnologias nativas da nuvem, também conhecidas como a *stack* nativa da nuvem, são as tecnologias usadas para criar [aplicativos nativos da nuvem](/pt-br/cloud_native_apps/). Essas tecnologias permitem que as organizações criem e executem aplicações escaláveis em ambientes modernos e dinâmicos, tais como as nuvens públicas, privadas e  híbridas, ao mesmo tempo em que aproveitam ao máximo os benefícios da [computação em nuvem](/cloud_computing/). Eles são projetados desde o início para explorar os recursos de computação em nuvem e contêineres, serviços em malha, microsserviços e infraestrutura imutável exemplificam essa abordagem.
```
Comment (jihoon-seo) [inline content/pt-br/cloud-native-tech.md:9]: @edsoncelio Could you check all the links and update if needed? 😊

- Underscores -> hyphens
- Update links to the localized ones if exist.
- (FYI: You can see the localized files in https://github.com/cncf/glossary/tree/dev-pt/content/pt-br )

```suggestion
As tecnologias nativas da nuvem, também conhecidas como a *stack* nativa da nuvem, são as tecnologias usadas para criar [aplicativos nativos da nuvem](/pt-br/cloud-native-apps/). Essas tecnologias permitem que as organizações criem e executem aplicações escaláveis em ambientes modernos e dinâmicos, tais como as nuvens públicas, privadas e  híbridas, ao mesmo tempo em que aproveitam ao máximo os benefícios da [computação em nuvem](/pt-br/cloud-computing/). Eles são projetados desde o início para explorar os recursos de computação em nuvem e contêineres, serviços em malha, microsserviços e infraestrutura imutável exemplificam essa abordagem.
```

**Review by jessicalins — APPROVED** [review]
LGTM :tada:

**Pull request merged**